packages/Babylonian-Tests.package/BPPrintbuggerTest.class/instance/setUp.st

```diff
@@ -0,0 +1,5 @@
+running
+setUp
+	
+	printbugger := BPPrintbugger new
```

Comment:
Probably, you have to add here:

```Smalltalk
{
	(BPTestManuallyInstrumentedClass>>#methodAWithAssertionUsingExpressionResult).
	(BPTestManuallyInstrumentedClass>>#methodWithMultipleProbes)} do: [:aMethod | 
		aMethod removeProperty: #bpExamples ifAbsent: []].
```

This will remove any remaining BPExamples from the #bpExamples property and will ensure a clean setup. You might also want to add this to tearDown!